### ISSUE TITLE
Connection to assetsd was interrupted or assetsd died

### ISSUE THREAD

**Issue by apexskier** [matthewpalmer/Regift#19]
I'm getting this error when using Regift. I think it's related to memory issues writing too large of a gif, but I'm wondering if anyone's seen or resolved this.

**Comment by matthewpalmer** [comment]
Hmm that's a really annoying error! I haven't come across it, so I'm not sure I can help fix it sorry. I will leave this open for a couple of weeks—hopefully someone else will come across it :)

**Comment by apexskier** [comment]
Yeah, it's not your fault. Turns out the CGImageDestinationFinalize loads all images in the GIF into memory at once and writes in one giant operation. If you have too many/too large images you'll run out of memory.

See http://stackoverflow.com/questions/29723024/creating-a-large-gif-with-cgimagedestinationfinalize-running-out-of-memory for more information.
